indico/core/signals/event/registration.py

```diff
@@ -159,3 +159,8 @@
 Called when the Pass object for an Apple Wallet has been generated. The `sender` is the
 `Registration` object, the `obj` kwarg contains the Pass object.
 ''')
+
+custom_ticket_qr_code_handler = _signals.signal('custom-ticket-qr-code-handler', '''
+Expected to return a subclass of ``CustomTicketCode`` implementing a custom ticket
+QR code handler. This can be used to handle custom ticket QR codes.
```

Comment:
Maybe we could add a warning or a disclaimer somewhere that the regex must be a JS regex and not a Python one? They support mostly the same features, but there are differences.

Reply:
It's already mentioned in the docstring of the regex arg in `CustomTicketCode`